pages/[resourceType]/[id]/index.tsx

```diff
@@ -63,11 +84,14 @@ function ResourceIDPage() {
           variant="filled"
           style={{
             float: "right",
+            marginRight: "8px",
+            marginLeft: "8px",
           }}
         >
           <div> Update </div>
         </Button>
       </Link>
+      {evaluateMeasureButton}
```

Comment:
Instead of rendering the state variable here, couldn't we just conditionally render the button based on the resourceType? i.e.:


``` tsx
{resourceType === "Measure" && (
  <Link href={`/${resourceType}/${id}/evaluate`} key={`evaluate-measure-${id}`} passHref>
    {/* rest of the button */}
)}
```

This eliminates the need for the state variable/useEffect setting of it entirely